src/kudu/security/security_flags.cc

```diff
@@ -27,14 +27,14 @@ namespace security {
 // below. The DH AES ciphers are not included since we are not configured to
 // use DH key agreement.
 const char* const SecurityDefaults::SecurityDefaults::kDefaultTlsCiphers =
```

Comment:
Not sure we cna combine the tls1.3 ciphers in the same list, you have to call SSL_CTX_set_ciphersuites vs SSL_CTX_set_cipher_list for 1.3 vs 1.2

Reply:
Backed out TLSv1.3 ciphers.